packages/query-core/src/tests/query.test.tsx

```diff
@@ -794,8 +794,12 @@ describe('query', () => {
     })
 
     const unsubscribe = observer.subscribe(() => undefined)
+
     await sleep(10)
-    expect(mockLogger.error).toHaveBeenCalledWith('Missing queryFn')
+    const query = queryCache.find(key)!
+    expect(mockLogger.error).toHaveBeenCalledWith(
+      `Missing queryFn for queryKey '${query.queryHash}'`,
```

Comment:
@TkDodo question here: wondering if testing the hash is the right thing to do here. It is an implementation detail. What about testing partially on `Missing queryFn for queryKey`?

WDYT?
Thanks.

Reply:
just theoretically, the test would then pass even if we don't put the hash in there. I think testing for it is fine, even with a different hashFn, you would see the desired result